views/home/updates.php

```diff
@@ -347,7 +353,7 @@ function issue($nr) {
 
 
 <dt><a name='x_2_4_1'></a>[2016-08-02] &mdash; Xdebug 2.4.1</dt>
- 
+
```

Comment:
In the future, I would like this to not be hard coded this way, and instead use the file based approach that I now use for news updates. But... that's not right now yet.

Reply:
This is also in the original files, right?

Reply:
yes!